### PR TITLE
Add maf to upload config, add sorting

### PR DESCRIPTION
**Pull request by 5im1z**

### CODE REVIEW

File: depmapomics/dm_omics.py [hunk 1]
```diff
@@ -1025,7 +1030,7 @@ def assign_end_pos(
                     "encoding": "utf-8",
                 },
                 {
-                    "path": folder + "somatic_mutations_profile.maf.txt",
+                    "path": folder + "somatic_mutations_profile.maf.csv",
```
Comment (qinqian) [inline depmapomics/dm_omics.py:1033]: Maf.csv suffix is not supported by IGV.

**Review by qinqian — CHANGES_REQUESTED** [review]
Could you please change the suffix of the maf file.

File: depmapomics/dm_omics.py (hunk 1 above)
Comment (5im1z) [reply depmapomics/dm_omics.py:1033]: This file should be reformatted to .txt in the subset & upload step downstream. Here I think we should keep it as a .csv so taiga can recognize it as a data frame, making it easier for subsetting

Comment (qinqian) [reply depmapomics/dm_omics.py:1033]: I agree if taiga has such a format requirement.

**Review by qinqian — APPROVED** [review]

**Pull request merged**